packages/uicore/webpack.config.js

```diff
@@ -56,7 +56,7 @@ module.exports = {
               sourceMap: isDev,
               modules: {
                 mode: 'local',
-                localIdentName: '[name]--[local]'
+                localIdentName: isDev ? '[name]--[local]' : '[name]--[local]--[hash:base64:5]'
```

Comment:
Don't do this. Both modes should produce the same. If some cases when you have to target/override from UICore, this change might cause huge issue as the output is diff when you develop vs. prod build.

Just keep the same hash.

Reply:
got it